projects/Terminal_Based_Hangman_Game/hangman.py

```diff
@@ -0,0 +1,180 @@
+import random
+import json
```

Comment:
import specific function from module
```suggestion
from json import load
```